src/g_editor.c

```diff
@@ -1900,10 +1905,10 @@ void canvas_vis(t_canvas *x, t_floatarg f)
                 sprintf(winpos, "+%d+%d", (int)(x->gl_screenx1), (int)(x->gl_screeny1));
             }
 
-            pdgui_vmess("pdtk_canvas_new", "^ ii si", x,
+            pdgui_vmess("pdtk_canvas_new", "^ ii si s", x,
                 (int)(x->gl_screenx2 - x->gl_screenx1),
                 (int)(x->gl_screeny2 - x->gl_screeny1),
-                winpos, x->gl_edit);
+                        winpos, x->gl_edit, x->gl_wm_state->s_name);
```

Comment:
please do not introduce any trailing whitespace.
(even better: actively remove any trailing whitespace in the files you touch; use a separate commit for whitespace-fixes)

Reply:
My bad I missed that one, my editor default was to align argument list to function parenthesis, I corrected it. Will add a new commit for fixing this one.